migrations/20191105143016_build-cars-table.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function(knex) {
+  return knex.schema.createTable("cars", table => {
+    table.increments();
+    table
+      .text("vin", 128)
+      .unique()
+      .notNullable();
+    table.text("make", 128).notNullable();
+    table.text("model", 128).notNullable();
+    table.float("mileage");
```

Comment:
add unsigned() if you want to prevent for the database to accept a negative number